### ISSUE TITLE
Using V1, +config.ts causes a blank '[build' file to generate on run

### ISSUE THREAD

**Issue by Ericarthurc** [brillout/vite-plugin-ssr#800]
### Description

Any directory that contains a `+config.ts` file using the V1 setup will have a `[build` file generate when starting dev mode or building the application.

![image](https://user-images.githubusercontent.com/22304079/233543005-a2eefd77-e9f5-4aa9-9e11-d04e83a794bd.png)

vite: 4.3.1
vite-plugin-ssr: 0.4.115

**Comment by brillout** [comment]
Are you using windows?

**Comment by Ericarthurc** [comment]
Yes windows 11
Using pnpm
Node 19

**Comment by brillout** [comment]
Replace the line `104` of `node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/transpileAndLoadPlusFile.js`:
```diff
-    const tag = `[build:${(0, utils_1.getRandomId)(12)}]`;
+    const tag = `[build-${(0, utils_1.getRandomId)(12)}]`;
```

Does it work then?

**Comment by Ericarthurc** [comment]
> Replace the line `104` of `node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/transpileAndLoadPlusFile.js`:
> 
> ```diff
> -    const tag = `[build:${(0, utils_1.getRandomId)(12)}]`;
> +    const tag = `[build-${(0, utils_1.getRandomId)(12)}]`;
> ```
> 
> Does it work then?

Yes changing the colon to a hyphen stoped the [build file from generation, and everything still seems to work fine

**Comment by brillout** [comment]
Released as `vite-plugin-ssr@0.4.115-draft.49055`.

**Comment by brillout** [comment]
Btw, in case I didn't mention it already and in case you think you can (eventually) convince your employer: https://github.com/sponsors/brillout 🙂.
